blueocean-core-js/src/js/components/ReplayButton.jsx

```diff
@@ -83,7 +83,7 @@ export class ReplayButton extends Component {
         return (
             <div className={`replay-button-component ${outerClass}`} onClick={(event => stopProp(event))}>
                 <a className={`replay-button ${innerButtonClass}`} title={replayLabel} onClick={() => this._onReplayClick()}>
-                    <Icon size={24} icon="replay" />
+                    <Icon size={24} icon="AvReplay" />
```

Comment:
Ok, looks like maybe it's the "category" from https://material.io/icons/   But aren't the icon names unique across all categories? Some quick searches seemed to indicate so. If that's the case, I would just use the icon name, probably all lowercase with hyphen or underscore as a delimiter.